src/midi/midi_fluidsynth.cpp

```diff
@@ -44,6 +42,149 @@
 
 constexpr auto SoundFontExtension = ".sf2";
 
+/**
+ * Platform specific extension for shared libraries
+ */
+#if defined(WIN32)
+#define DB_FLUID_LIB_EXT ".dll"
+#else
+#define DB_FLUID_LIB_EXT ".so"
```

Comment:
Not entirely sure whether it's `.so` or `.dylib` on the Mac in this case. Can't test on a Mac now, so maybe @kklobe will be able to confirm.